spec/models/events/review_spec.rb

```diff
@@ -51,4 +51,28 @@
       expect(subject).to_not be_valid
     end
   end
+
+  describe 'callbacks' do
+    context 'when a review is created' do
+      let(:review) { create :review, review_request: review_request }
+      let(:review_request) { create(:review_request) }
+
+      it 'triggers review turnaround creation' do
+        expect_any_instance_of(ReviewTurnaround).to receive_message_chain(:set_value, :save!)
```

Comment:
this test is just testing that a method is called, it currently works but IMHO it's heavily tied to implementations details. I think a test like the following will be actually checking what you really care about

```ruby
it 'creates a review turnaround with the correct values' do
  expect { review }.to change { ReviewTurnaround.count }.from(0).to(1)

  review_turnaround = ReviewTurnaround.last
  expect(review_turnaround.[some_attribute]) = some_expected_value
  ...
  expect(review_turnaround.[some_attribute]) = some_expected_value
end
```